randovania/data/json_data/prime3.txt

```diff
@@ -1963,13 +2310,19 @@ Asset id: 15414731845526197010
 > Dock to Corrupted Pool; Heals? False
   * Normal Door to Corrupted Pool/Dock to Gel Processing Site
   > Dock to Gel Hall
+      After Gel Processing Site Pillar
+  > Event - Raise Pillar
       Grapple Lasso
 
 > Dock to Gel Hall; Heals? False
   * Normal Door to Gel Hall/Dock to Gel Processing Site
   > Dock to Corrupted Pool
       Trivial
 
+> Event - Raise Pillar; Heals? False
+  > Dock to Corrupted Pool
+      Trivial
+
 ----------------
 Save Station A
 Asset id: 6190253681364271262
```

Comment:
Add Node called "Save Station" with "Heals?" checked, and have it connect to either door node. Have the door nodes only connect to the Save Station node.